modules/widgets/social-media-icons.php

```diff
@@ -25,16 +25,17 @@ public function __construct() {
 		);
 
 		$this->defaults = array(
-			'title'              => __( 'Social', 'jetpack' ),
-			'facebook_username'  => '',
-			'twitter_username'   => '',
-			'instagram_username' => '',
-			'pinterest_username' => '',
-			'linkedin_username'  => '',
-			'github_username'    => '',
-			'youtube_username'   => '',
-			'vimeo_username'     => '',
+			'title'               => __( 'Social', 'jetpack' ),
+			'facebook_username'   => '',
+			'twitter_username'    => '',
+			'instagram_username'  => '',
+			'pinterest_username'  => '',
+			'linkedin_username'   => '',
+			'github_username'     => '',
+			'youtube_username'    => '',
+			'vimeo_username'      => '',
 			'googleplus_username' => '',
+			'wordpress_username'  => '',
```

Comment:
I ended up reverting the switch to `wordpressorg`, sorry about that! I originally didn't think of the consequences when suggesting renaming to `wordpressorg`.
While it is more specific, it then breaks the display of the WordPress logo
since Genericons use the `wordpress` class to display the logo.

Let's then have WordPress.org be the default `wordpress`.

I've tested it, it works well. LGTM! 🚢